### PR TITLE
Dev -> master

### PR DESCRIPTION
**Pull request by kris-watts-gravwell**
for 5.2.1 release

### CODE REVIEW

**Comment by ashnwade** [comment]
We should also include the updates to third party licenses before we merge this.

**Comment by kris-watts-gravwell** [comment]
ok, we'll get those in on another PR and then you can approve and merge this when you are ready.

**Review by david-fritz-gravwell — APPROVED** [review]
LGTM

**Pull request merged**
